src/cdk/drag-drop/drag.ts

```diff
@@ -218,6 +227,13 @@ export class CdkDrag<T = any> implements AfterViewInit, OnDestroy {
    */
   @Input('cdkDragRootElement') rootElementSelector: string;
 
+  /**
+   * Selector that will be used to determine the element to which the draggable's position will
+   * be constrained. Matching starts from the element's parent and goes up the DOM until a matching
+   * element has been found.
+   */
+  @Input('cdkDragBoundry') boundryElementSelector: string;
```

Comment:
Is it really called `boundry`? Please correct me if I'am wrong, but I thought it's `boundary`

Reply:
It's boundary, good catch.